### ISSUE TITLE
CI version numbers

### ISSUE THREAD

**Issue by modulesio** [webmixedreality/exokit#113]
Would be great to put version numbers in the CI uploads.

E.g. `exokit-win-x64-0.0.308`

#### Problems

- It's very CI-specific to upload a name based on an env var (version number)
- https://get.webmr.io/ would need to be aware of this so it picks up the version-decorated asset

**Comment by shawwn** [comment]
I think this is a symptom of a more general problem: It's not a good idea to cut releases so often. And when you do, it's probably fine to manually name them.

**Comment by modulesio** [comment]
Probably will be much more manual with post 1.0...

**Comment by modulesio** [comment]
This was recently asked for again.

**Comment by chrisplatorres** [comment]
We are no longer doing CI uploads, only installers or packages, which might make sense for packages... but the goal is to make the desktop Exokit launchers self-updating.

**Comment by modulesio** [comment]
True, if this is an option in the studio, then version numbers on the file names won't matter.
